plugin-flex-ts-template-v2/src/feature-library/keyboard-shortcuts/flex-hooks/strings/index.ts

```diff
@@ -0,0 +1,98 @@
+  ModalSetShortcutErrorMsg = 'KeyboardMgrModalSetShortcutErrorMsg',
+  SettingsResetShortcuts = 'KeyboardMgrResetShortcuts',
+  SettingsResetShortcutsText = 'KeyboardMgrResetShortcutsText',
+  SettingsResetShortcutsTitle = 'KeyboardMgrResetShortcutsTitle',
+  SettingsRemoveShortcuts = 'KeyboardMgrRemoveShortcuts',
+  SettingsRemoveShortcutsText = 'KeyboardMgrRemoveShortcutsText',
+  SettingsDeleteIndividualShortcuts = 'KeyboardMgrDeleteIndividualShortcuts',
+  SettingsDeleteIndividualShortcutsText = 'KeyboardMgrDeleteIndividualShortcutsText',
+  SettingsEnableKeyThrottling = 'KeyboardMgrEnableKeyThrottling',
+  SettingsEnableKeyThrottlingText = 'KeyboardMgrEnableKeyThrottlingText',
+  SettingsToastDisableNotification = 'KeyboardMgrToastDisableNotification',
+  SettingsToastResetNotification = 'KeyboardMgrToastResetNotification',
+}
+
+export const stringHook = () => ({
+  'en-US': {
+    [StringTemplates.DefaultTitle]: 'Default keyboard shortcuts',
+    [StringTemplates.CustomTitle]: 'Custom keyboard shortcuts',
+    [StringTemplates.SettingsTitle]: 'Keyboard shortcuts settings',
+    [StringTemplates.MainTitle]: 'My Shortcut Settings',
+    [StringTemplates.HeaderModifiers]: 'Modifiers',
+    [StringTemplates.HeaderShortcuts]: 'Shortcuts',
+    [StringTemplates.HeaderActions]: 'Actions',
+    [StringTemplates.HeaderThrottle]: 'Throttle (ms)',
+    [StringTemplates.HeaderEdit]: 'Edit',
+    [StringTemplates.NotConfiguredMsg]: 'Not configured',
+    [StringTemplates.ErrorMsgNoDefaultShortcuts]:
+      'There are no configured keyboard shortcuts. Please reset your keyboard settings to enable keyboard shortcut customization.',
+    [StringTemplates.ErrorMsgNoCustomShortcuts]:
+      '   There are no configured custom keyboard shortcuts. Please reset your keyboard settings to enable custom keyboard shortcuts.',
+    [StringTemplates.WarningMsg]: 'Re-enable Keyboard Shortcuts',
+    [StringTemplates.TooltipMsg]: 'Ctrl and Shift are the default modifiers that cannot be changed.',
+    [StringTemplates.ToasterDeleteNotification]:
+      'Keyboard shortcut named {{actionName}} has been successfully deleted.',
+    [StringTemplates.ToasterSuccessNotification]:
+      'Keyboard action {{actionName}} modified successfully from {{oldShortcut}} to {{upperCase}}! Your new keyboard shortcut is: Ctrl + Shift + {{upperCase}},',
+    [StringTemplates.ModalTitle]: 'Modify a keyboard shortcut',
+    [StringTemplates.ModalCurrentConfig]: 'Current configuration',
+    [StringTemplates.ModalNewConfig]: 'New configuration',
+    [StringTemplates.ModalHeaderAction]: 'Action name',
+    [StringTemplates.ModalHeaderCurrent]: 'Current shortcut',
+    [StringTemplates.ModalInputLabel]: 'New keyboard shortcut',
+    [StringTemplates.ModalThrottleInputLabel]: 'Throttle',
+    [StringTemplates.ModalHelpText]: 'Enter your new keyboard shortcut',
+    [StringTemplates.CancelButton]: 'Cancel',
+    [StringTemplates.SaveButton]: 'Save',
```

Comment:
Flex UI has built-in translations for these two (`templates.Save` and `templates.Cancel`) so we can reduce translation effort by using those.

Reply:
Got it; updated.